phpunit.xml.dist

```diff
@@ -0,0 +1,23 @@
+<phpunit
+	backupGlobals="false"
+	bootstrap="tests/phpunit/bootstrap.php"
+	colors="true"
+	convertErrorsToExceptions="true"
+	convertNoticesToExceptions="true"
+	convertWarningsToExceptions="true">
+
```

Comment:
Could add a [default test suite](https://github.com/GaryJones/plugin-boilerplate/blob/4c3fc9ec57c35257c9293dfbd74030d72cb4e0b2/phpunit.xml.dist#L9).

Reply:
I tried that out when experimenting with different ways to load the test suites, but don't remember it offering any benefit that I could think of. `composer test` already sets the test suite to `Unit`, so I figured calling `./vendor/bin/phpunit` should just run them all.

It also allows for running something like `./vendor/bin/phpunit --filter=ThemeTest` without having to add `--testsuite=Integration`.